templates/account.html

```diff
@@ -10,8 +10,8 @@
 <h1>Log In:</h1>
 <form action="/user/login" method="POST">
     <font color='red' id='msg2'></font><br>
-    <label>Username: </label><input type="text" name="handle" class="form-control"/><br/>
-    <label>Password: </label><input type="password" name="password" class="form-control"/><br/>
+    <label>Username: </label><input type="text" name="handle" class="form-control" minlength="1" maxlength="20"/><br/>
+    <label>Password: </label><input type="password" name="password" class="form-control" minlength="4" maxlength="20"/><br/>
```

Comment:
Are you validating on these lengths on the server-side? I'll show Dalton how to send a `POST` using the `requests` module or with `curl`.